doc/nrf/releases/release-notes-1.6.0.rst

```diff
@@ -0,0 +1,658 @@
+* Updated:
+
+  * Reworked the :ref:`NCP sample <zigbee_ncp_sample>` to work with the simple gateway application.
+  * Moved the `NCP Host documentation`_ from the `ZBOSS NCP Host`_ package to the same location as the `external ZBOSS development guide and API documentation`_.
+
+nRF Desktop
+-----------
+
+* Fixed issues with boot reports on the USB backend.
+* Adapted the application to use common modules from the :ref:`lib_caf` library.
+* Fixed minor bugs.
+
+Common
+======
+
+The following changes are relevant for all device families.
+
+Trusted Firmware-M
+------------------
+
+* Added support for hardware-accelerated cryptography in TF-M using the Nordic Security module (nrf_security).
+  When enabled (default), any calls to psa_crypto APIs will utilize the CryptoCell hardware on nRF9160 and nRF5340.
+* Added support for using hardware unique keys (HUKs) for key derivation (``TFM_CRYPTO_ALG_HUK_DERIVATION``).
+  TF-M automatically generates and stores random hardware unique keys (if not present), using the :ref:`lib_hw_unique_key` library.
+* TFM_MINIMAL: Added a size-optimized configuration of TF-M which provides a minimal set of features:
+
+  * This configuration requires 32 KB of flash and provides random number generation, SHA-256, and the platform memory read service, which is analogous to the feature set of :ref:`secure_partition_manager`.
+  * The configuration is showcased in the :ref:`tfm_hello_world` sample.
+
+* The TF-M protected storage service is now using non-static labels when deriving encryption keys.
+  The derivation labels are generated by combining the client ID of the requesting service and the UID of the resource.
+* The TF-M protected storage on nRF9160 is now configured to use AES CCM to perform authenticated encryption and decryption.
+* You can now run PSA tests (psa-arch-tests) and TF-M regression tests in the |NCS|.
+  The tests can be found as samples in :file:`zephyr/samples/tfm_integration`.
+
+Crypto
+------
+
+* :ref:`crypto_samples`:
+
+  * Added samples showcasing the usage of the Platform Security Architecture (PSA) Crypto APIs.
+    The samples perform various cryptographic operations such as encryption/decryption using symmetric and asymmetric ciphers, key exchange, hashing, and random number generation.
+    Both TF-M enabled targets and secure-only targets are supported.
+
+* :ref:`lib_hw_unique_key` library:
+
+  * New library for managing and using hardware unique keys (HUKs), building on the APIs in nrf_cc3xx_platform.
+    HUKs are secret keys that are kept hidden from the application code, but which can be used by the application for deriving keys for other purposes, such as encrypting data for storing.
+
+Edge Impulse
+------------
+
+* Added the :ref:`nrf_machine_learning_app` application that integrates the Edge Impulse wrapper library with sensor sampling.
+* Sample reference to a public pre-trained model.
```

Comment:
I was sure I commented on this line, but looks like I haven't:
```suggestion
Added the :ref:`nrf_machine_learning_app` application that integrates the Edge Impulse wrapper library and the Edge Impulse data forwarder with sensor sampling.
```
Discussed and approved with @MarekPieta

Reply:
Sorry @greg-fer  @MarekPieta  but it's too late for this change to go in